applications/tari_app_grpc/proto/wallet.proto

```diff
@@ -39,6 +42,13 @@ service Wallet {
 
 message GetVersionRequest { }
 
+message GetIdentityRequest { }
+
+message GetIdentityResponse {
+    string public_key = 1;
```

Comment:
```suggestion
    bytes public_key = 1;
```

Reply:
Done, added node_id of type bytes as well.